lib/builder.js

```diff
@@ -264,7 +264,7 @@ function executeConfigFile(saveForReset, ignoreBaseURL, source) {
   };
 
   // jshint evil:true
-  new Function(source.toString()).call(global);
+  eval(source.toString());
```

Comment:
One possibility is to shim `System` in the `new Function` call. Eg:

```js
new Function(`
    var System = { confs: [], config: function (conf) { this.confs.push(conf); } };
    ${source.toString()}
    return System;
`).call(global);
```
The System shim would need be to be fleshed out much more and it would break if the config file had a `return` in it...